api-editor/gui/src/features/packageData/treeView/TreeNode.tsx

```diff
@@ -65,4 +82,11 @@ const isSelected = function (declaration: PythonDeclaration, currentPathname: st
     return `/${declaration.pathAsString()}` === currentPathname;
 };
 
+const getColorFromValue = function (maxValue: number, specificValue: number): string {
+    const percentage = Math.log(specificValue) / Math.log(maxValue);
```

Comment:
Not quite correct:

|`specificValue`|`maxValue`|desired result|actual result|
|-|-|-|-|
|0|0|0|NaN|
|0|10|0|-Infinity|
|1|1|1|NaN|
|1|2|0<x<1|0|

Suggestion:

```text
If specificValue <= 0: return 0
Else if specificValue >= maxValue: return 1
Else: return log(specificValue + 1) / log(specificValue + 1)
```

The conditions `specificValue < 0` and `specificValue >= maxValue` are a precaution.

Reply:
Has been implemented.
As a precaution the parameters are tested for impossible values.
Also the base values for the logarithmic calculation has been changed from 0 to 1, which should fix the third and fourth issue.